nucleus/model.py

```diff
@@ -92,22 +92,24 @@ class Model:
     endpoint, using :meth:`NucleusClient.add_model`.
     """
 
-    def __init__(self, model_id, name, reference_id, metadata, client):
+    def __init__(self, model_id, name, reference_id, metadata, bundle_id, client):
```

Comment:
Should we have a default `bundle_id=None` or is that something that Nucleus doesn't do? I guess if this isn't ever user-created it's probably fine to not have this default?